docs/pmm-upgrade/upgrade_podman.md

```diff
@@ -1,63 +1,52 @@
-# Upgrade PMM Server using podman
+# Upgrade PMM Server using Podman
 
-??? info "Summary"
+## Before you begin
 
-    !!! summary alert alert-info ""
-        - Perform a backup.
-        - Update PMM tag.
-        - Pre-pull image.
-        - Run it.
-
-    ---
+Before starting the upgrade, complete these preparation steps to ensure you can recover your system if needed and confirm compatibility with the new version:
+{.power-number}
 
-!!! caution alert alert-warning "Important"
-    You cannot downgrade. To go to a previous version, you must create a backup before upgrading.
+1. Create a backup before upgrading, as downgrades are not possible. Therefore, reverting to a previous version requires an backup made prior to the upgrade.
 
-!!! hint alert alert-success "Tip"
-    To see the current release running on your system, use the *PMM Upgrade* panel on the *Home Dashboard*, or run:
+2. Verify your current PMM version: Check your current PMM version by navigating to **PMM Configuration > Updates** or by running the following command. If accessing remotely, replace `localhost` with the appropriate IP address or server name:
```

Comment:
Same as with the docker instructions, I think when executing the curl command inside the container, the ip/domain will be localhost (since you are executing the command in the same place where the pmm server is running).